src/tox/session/__init__.py

```diff
@@ -289,3 +291,14 @@ def showconfig(self):
 
     def showenvs(self, all_envs=False, description=False):
         show_envs(self.config, all_envs=all_envs, description=description)
+
+    def make_devenv(self):
+        # envdir implies --notest
+        self.config.option.notest = True
```

Comment:
Let's move all this to the end of the configuration in config module, and let the current mechanism create the env. This way we don't need a separate execution target and things will fail and show correctly with ``--showconfig`` (which never hits this).

Reply:
ah yeah that simplifies things greatly -- will do!